huaweicloud/services/modelarts/data_source_huaweicloud_modelarts_notebook_images.go

```diff
@@ -0,0 +1,153 @@
+package modelarts
+
+import (
+	"context"
+
+	"github.com/chnsz/golangsdk/openstack/modelarts/v1/notebook"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/hashcode"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+func DataSourceNotebookImages() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceNotebookImagesRead,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"organization": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"type": {
```

Comment:
can we set the default value to `BUILD_IN`?

Reply:
ok